container_images/tox/README.md

```diff
@@ -0,0 +1,63 @@
+A prow worker that executes Python tests using [tox](https://tox.readthedocs.io/).
+Check the Dockerfile for which Python versions are installed.
+
+### Interface
+
+Your job:
+1. Create a [tox configuration file](https://tox.readthedocs.io/en/latest/config.html) in your
+repository's test directory.
+   * Optionally, write test artifacts to the `ARTIFACTS` directory.
+1. Configure prow with the test directory's path as the first positional argument.
+
+Prior to invoking, the runner will:
+1. Checkout your repository, mount it as a volume, and set its root as Docker's 
+[workdir](https://docs.docker.com/engine/reference/commandline/run/#set-working-directory--w).
+2. Mount a volume for artifacts, and inject its path with the `ARTIFACTS` env variable.
```

Comment:
all volume mounting should be in the prow config. how will prow know to mount this additional volume?